rustls/src/verify.rs

```diff
@@ -99,6 +99,10 @@ pub trait ServerCertVerifier: Send + Sync {
     /// along with the end-entity certificate; it is in the same order that the
     /// peer sent them and may be empty.
     ///
+    /// Note that none of the certificates have been parsed yet, so it is the responsibility of
+    /// the implementor to handle invalid data. It is recommended that the implementor returns
+    /// [Error::InvalidCertificateEncoding] when these cases are encountered.
```

Comment:
Can we do an intra-doc link here? ```[`Error::InvalidCertificateEncoding`]```?

Reply:
I think that they already are, no? Rustdoc and VSCode seem to recognize it correctly:

![image](https://user-images.githubusercontent.com/68570223/171788665-2b760ee8-4bf4-4cf3-95c5-6c096d74fdc9.png)

However, I've updated both of them to use backticks so that they appear nicer when rendered.